core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7146,6 +7149,8 @@ components:
           $ref: "#/components/schemas/LedgerTransaction"
         receipt:
           $ref: "#/components/schemas/TransactionReceipt"
+        outcomes:
```

Comment:
I think "outcome" would be more appropriate (instead of `outcomes`) but even than I'm not sure of.

I think I'd either prefer (A) including the full LtsTransactionOutcome in this directly, or (B) just calling this balance changes, like we did in the Gateway.

I don't mind which, but I was never entirely happy with the name "outcome" and I'd rather not introduce another model which is different and also labelled as the outcome :D.